### PR TITLE
Feat/proxy cache debug

### PR DESCRIPTION
**Pull request by utix**
### Summary

Remove proxy cache headers from response if the kong debug is not set.
Adding Miss/Hit, Age and cache key to answer shouldn't be the default behavior.

### Full changelog

The following headers:
* X-Cache-Status
* X-Cache-Key
* Age

are now set only if the header kong-debug is set to 1

Refactor test to remove duplicated lines and use get, post and delete helpers instead of send

### CODE REVIEW

File: kong/plugins/proxy-cache/handler.lua [hunk 1]
```diff
@@ -356,8 +360,15 @@ function ProxyCacheHandler:access(conf)
     end
   end
 
-  res.headers["Age"] = floor(time() - res.timestamp)
-  res.headers["X-Cache-Status"] = "Hit"
+
+  if ngx.var.http_kong_debug then
```
Comment (locao) [inline kong/plugins/proxy-cache/handler.lua:364]: Not a blocker, but you could use `set_header` here, right?

**Review by locao — APPROVED** [review]

File: kong/plugins/proxy-cache/handler.lua (hunk 1 above)
Comment (utix) [reply kong/plugins/proxy-cache/handler.lua:364]: It was not clear why it was not the case before, so I have prefered to keep it as it. I will rebase on 2.7.0 and do some tests to check

Comment (utix) [reply kong/plugins/proxy-cache/handler.lua:364]: I get it, on the function `resè is retrieved from the cache
`  local res, err = strategy:fetch(cache_key)`
`res` is updated and answered with `  return kong.response.exit(res.status, res.body, res.headers)`
We have to set res.hearders we cannot use the helper here.

**Comment by hanshuebner** [comment]
@utix Sorry for the very late reply.  Are you still interested in this PR?  If so, would you mind rebasing it to master?

**Comment by hanshuebner** [comment]
We're reverting this PR (https://github.com/Kong/kong/pull/10444) as it seems to cause our tests to fail.  Please reopen if it is still needed.

**Comment by utix** [comment]
@hanshuebner I rebase an fix the tests #10445

**Pull request merged**